### PR TITLE
[AP-1060] Use orjson

### PR DESCRIPTION
**Pull request by koszti**
# Description of change

Taps and targets are using python-builtin and simplejson JSON libraries. We'd like to improve the overall performance of [tap-kafka](https://github.com/transferwise/pipelinewise-tap-kafka) and the JSON operations are done by this package.

This PR is refactoring the code to use [orjson](https://github.com/ijl/orjson) which overperforms and benchmarked as the fastest JSON library:
* https://github.com/ijl/orjson#performance
* https://github.com/ultrajson/ultrajson#benchmarks
* https://levelup.gitconnected.com/json-encoding-decoding-with-python-62a2cae63a6a

# Manual QA steps
 - 
 
# Risks
 - 
 
# Rollback steps
 - revert this branch

### CODE REVIEW

File: singer/schema.py [hunk 1]
```diff
@@ -57,7 +57,7 @@ def __init__(self, type=None, format=None, properties=None, items=None,
         self.patternProperties = patternProperties
 
     def __str__(self):
-        return json.dumps(self.to_dict())
+        return orjson.dumps(self.to_dict())
```
Comment (marksuhhin) [inline singer/schema.py:60]: Orjson returns back bytes, instead of string, so it would make sense to cast here to string before return.

Comment (koszti) [reply singer/schema.py]: thanks, fixed by 5c272cfc82e75dec9d2e9b9b6b6086785cc90a1f

**Comment by Samira-El** [comment]
Have you tried running these changes from within one of the connectors to see how it behaves?

**Review by louis-pie — APPROVED** [review]

**Comment by koszti** [comment]
> Have you tried running these changes from within one of the connectors to see how it behaves?

@Samira-El , yes, I tested it with the following connectors by pointing their dependency to this branch and running the existing unit and integrations test in the connectors. All of them are passing by using this branch:
* pipelinewise-tap-kafka
* pipelinewise-tap-mysql
* pipelinewise-tap-postgres
* pipelinewise-target-snowflake

**Comment by Samira-El** [comment]
Cool

**Review by Samira-El — APPROVED** [review]

**Pull request merged**